### PR TITLE
Create PRS13969GabraZaraSyon.xml

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
Created new PersonID for owner in BMLacq681

### CODE REVIEW

File: new/PRS13969GabraZaraSyon.xml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        </profileDesc>
+        <revisionDesc>
+            <change who="" when="2023-01-26">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person>
+                    <persName xml:lang="gez" xml:id="n1">Owner</persName>
```
Comment (eu-genia) [inline new/PRS13969GabraZaraSyon.xml:48]: you need the name in fidel here

File: new/PRS13969GabraZaraSyon.xml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+                    <persName xml:lang="gez" type="normalized" corresp="#n1"/>
```
Comment (eu-genia) [inline new/PRS13969GabraZaraSyon.xml:49]: you need the name in transcription here

File: new/PRS13969GabraZaraSyon.xml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+                    <faith type="EOTC"/>
+                    <birth/>
+                    <death/>
+                    <floruit>17th or 18th centuries</floruit>
+                </person>
+            </listPerson><!---->
```
Comment (eu-genia) [inline new/PRS13969GabraZaraSyon.xml:55]: you need a relation specifying that this is an owner of BMLacq681

File: new/PRS13969GabraZaraSyon.xml [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person>
+                    <persName xml:lang="gez" xml:id="n1">Owner</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1"/>
+                    <faith type="EOTC"/>
+                    <birth/>
```
Comment (eu-genia) [inline new/PRS13969GabraZaraSyon.xml:51]: you can remove empty elements

File: new/PRS13969GabraZaraSyon.xml [hunk 5]
```diff
@@ -0,0 +1,58 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS13969GabraZaraSyon" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Owner</title>
```
Comment (eu-genia) [inline new/PRS13969GabraZaraSyon.xml:8]: the title should be the name of the person

File: new/PRS13969GabraZaraSyon.xml [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+                <editor role="generalEditor" key="AB"/>
+                <editor key=""/>
```
Comment (eu-genia) [inline new/PRS13969GabraZaraSyon.xml:10]: key="SH" ( I will try to check why this was not done by the app)

File: new/PRS13969GabraZaraSyon.xml [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="" when="2023-01-26">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person>
```
Comment (eu-genia) [inline new/PRS13969GabraZaraSyon.xml:47]: `<person sex="1">`
(the app would have done it if you had selected "male" in the form

**Review by eu-genia — CHANGES_REQUESTED** [review]
thank you!

File: new/PRS13969GabraZaraSyon.xml (hunk 6 above)
Comment (eu-genia) [reply new/PRS13969GabraZaraSyon.xml:10]: the initials are also missing in the `who `attribute of the `change `element

NB this is the reason the committed record was red in Oxygen, it should be green

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/PRS13969GabraZaraSyon.xml (hunk 1 above)
Comment (CarstenHoffmannMarburg) [reply new/PRS13969GabraZaraSyon.xml:48]: I understand, but I don't know, how to make these changes.

**Comment by eu-genia** [comment]
Dear Carsten, when you create a new record you are prompted to download it to the NEW folder, which you correctly did. Now you have the record in your computer. You should open it in Oxygen and edit. And then commit. Thank you!

**Comment by CarstenHoffmannMarburg** [comment]
I don't see the new records in the folder and I can't open it on Oxygen

File: new/PRS13969GabraZaraSyon.xml (hunk 1 above)
Comment (eu-genia) [reply new/PRS13969GabraZaraSyon.xml:48]: Another reminder to carry out changes

Comment (CarstenHoffmannMarburg) [reply new/PRS13969GabraZaraSyon.xml]: I cannot find the correct branch in Github or the file in Oxygen.

File: new/PRS13969GabraZaraSyon.xml (hunk 3 above)
Comment (CarstenHoffmannMarburg) [reply new/PRS13969GabraZaraSyon.xml:55]: Done.

**Review by eu-genia — APPROVED** [review]

**Pull request merged**